test/0.4.24/lidooracle.test.js

```diff
@@ -114,11 +109,30 @@ contract('LidoOracle', ([appManager, voting, user1, user2, user3, user4, nobody]
       await app.removeOracleMember(user2, { from: voting })
 
       await assertRevert(app.removeOracleMember(user2, { from: user1 }), 'APP_AUTH_FAILED')
-      await assertRevert(app.removeOracleMember(user3, { from: voting }), 'QUORUM_WONT_BE_MADE')
 
       assert.deepStrictEqual(await app.getOracleMembers(), [user3])
     })
 
+    it('removeOracleMember updates minReportableEpochId', async () => {
```

Comment:
It's critical to check that removeOracleMember clears all previous votes. Did't find this test.